analytics/lib/counts.py

```diff
@@ -1,5 +1,6 @@
 from django.db import connection, models
```

Comment:
s/changed/added -- all the analytics tables are insert only, and we don't want to suggest otherwise.